R/peak.characterize.R

```diff
@@ -9,8 +9,8 @@ function(existing.row=NA, ftrs.row, chr.row)
             if(n==1) x<-c(x,mean0)
             mean1<-mean(x)
             sd1<-sd(x)
-            min1<-min(x)
-            max1<-max(x)
+            min1<-min(x, Inf)
+            max1<-max(x, -Inf)
```

Comment:
What does this change actually cause?

Reply:
`min` on empty vector returns `Inf` and prints this warning:
```
Warning message:
In min(empty_vector) : no non-missing arguments to min; returning Inf
```

`max` does the same but returns `-Inf`.

By adding `Inf` and `-Inf` to the arguments, we omit these warnings while not changing a returned value.